### DISCUSSION TITLE
希望增加hypixel战绩查询功能！

### DISCUSSION THREAD

**Discussion by AMGYzhea** [FloatTech/ZeroBot-Plugin#677] [Ideas]
查询MC的**hypixel**服务器战绩功能
调用玩家自己上传的APIkey
这个功能好多机器人都实现了，期待ZeroBOT可以拥有
应该有不少群也需要这个功能......

**Comment by aiwandiannaodelele** [comment]
## 第一个帖子

这是综合讨论区的第一个帖子！欢迎来到 乐乐 Lab 的社区讨论区。

在这里你可以：
- 分享你的想法和发现
- 提问和寻求帮助
- 和大家交流技术话题
- 随意闲聊

目前站点还在建设中，后续会陆续更新更多内容。感谢你的到来！

---

*发自我的网站评论区*
